content/post/handshakes_pagerank.md

```diff
@@ -73,17 +73,17 @@ It is a pickled `scipy.sparse.csr_matrix`, the following Python code loads it:
 
 ```python
 import pickle
-with open("graph.pickle", "rb") as fin:
+with open("graph_blog_post.pickle", "rb") as fin:
     graph, repos = pickle.load(fin)
-ndevs = graph.shape[0] - len(repos)
-print(repr(graph))
-print("Number of developers:", ndevs)
-print("Number of repositories:", len(repos))
-inv_repos = {r: i + ndevs for i, r in enumerate(repos)}
-name = "src-d/go-git"
-print("Repository %s has %d contributors, %d commits" % (
-    name, len(graph[inv_repos[name]].indices), graph[inv_repos[name]].sum()
-))
+    ndevs = graph.shape[0] - len(repos)
```

Comment:
This is Python: variables do not have a local scope (function closures are present though). It is a common practice to read and close the file as soon as possible, and continue with the loaded objects. So no need for this extra indentation.